cmd/cli/uninstall_mesh.go

```diff
@@ -199,6 +182,63 @@ func (d *uninstallMeshCmd) run() error {
 			fmt.Fprintf(d.out, "OSM namespace CANNOT be deleted in a managed environment\n")
 		}
 	}
+
+	err := d.deleteClusterResources()
+	return err
+}
+
+func (d *uninstallMeshCmd) findFlagSpecifiedMesh(meshInfoList []meshInfo) (bool, bool) {
+	meshFlagSpecified := d.meshName != ""
+	specifiedMeshFound := false
+	if !meshFlagSpecified {
+		return meshFlagSpecified, specifiedMeshFound
+	}
+
+	specifiedMeshFound = d.findMesh(meshInfoList)
+	if !specifiedMeshFound {
+		fmt.Fprintf(d.out, "Did not find mesh [%s] in namespace [%s]\n", d.meshName, d.meshNamespace)
+		// print a list of meshes within the cluster for a better user experience
+		if err := d.printMeshes(); err != nil {
+			fmt.Fprintf(d.out, "Unable to list meshes in the cluster - [%v]", err)
+		}
+	}
+
+	return meshFlagSpecified, specifiedMeshFound
```

Comment:
again it's easier to hard code the value if we know its static, vs tracing to figure it out.

ie: meshFlagSpecified is guaranteed to be true here.

Although with above comment, we should be able to just return specifiedMeshFound